sdk/storage/storage-file-share/test/directoryclient.spec.ts

```diff
@@ -2464,3 +2471,132 @@ describe("DirectoryClient - AllowingTrailingDots - Default", () => {
     assert.equal(await dirClient1.exists(), false);
   });
 });
+
+describe("DirectoryClient - NFS", () => {
+  let recorder: Recorder;
+  let shareName: string;
+  let shareClient: ShareClient;
+  let dirName: string;
+  let dirClient: ShareDirectoryClient;
+
+  beforeEach(async function (this: Context) {
+    recorder = new Recorder(this.currentTest);
+    await recorder.start(recorderEnvSetup);
+    await recorder.addSanitizers({ uriSanitizers }, ["record", "playback"]);
+    let serviceClient: ShareServiceClient;
+    try {
+      serviceClient = getGenericBSU(recorder, "PREMIUM_FILE_");
+    } catch (error: any) {
+      console.log(error);
+      this.skip();
+    }
+
+    shareName = recorder.variable("share", getUniqueName("share"));
+    shareClient = serviceClient.getShareClient(shareName);
+    await shareClient.create({
+      protocols: {
+        smbEnabled: true,
+        nfsEnabled: true,
```

Comment:
Rest API should only allow put 1 protocal.

Reply:
Updated